climada_petals/hazard/coastal_flood.py

```diff
@@ -0,0 +1,134 @@
+from typing import Iterable, Union, Optional
+import numpy as np
+from shapely.geometry import Polygon
+
+from climada.hazard import Hazard
+from climada.util import files_handler as u_fh
+import climada.util.coordinates as u_coord
+from climada import CONFIG
+
+AQUEDUCT_SOURCE_LINK = CONFIG.hazard.coastal_flood.resources.aqueduct.str()
+# FIX THIS PATH - YOU SHOULD SAVE UNDER HAZARD, NOW IT CREATES A FOLDER CALLED COASTALFLOOD
+# AT THE SAME LEVEL AS HAZARD. RIVERINE AQUEDUCT IS SAVED THERE TOO, SO FIX THAT TOO
+# (SEE CLIMADA DATA LOCAL FOLDER)
```

Comment:
Is this comment still valid? Actually the path ~/climada/data/hazard is where the files from the data api land. There are some automatic features on this directory executed by the api client: caching, making sure that the version is always the newest and purging. So, strictly speaking, the data files that are not handled by the climada api client _should_ not necessarily be saved there. Any other directory on the same level actually seems better suited.

Reply:
removed